stylesheets/_component.panels.scss

```diff
@@ -60,9 +60,13 @@
 
 }
 
+%panel-icon-color {
+    color: color(white);
+}
+
 @each $state, $state-color, $state-color-alt in $state-colors {
     .panel--#{$state} .remove-button > [class^="icon-"] {
```

Comment:
At a first glance I'm wondering why we (well, me, cos I wrote it) aren't just doing `.panel .remove-button > [class^="icon-"]` seeing as it's being applied to all state variations including base 🤔